contracts/facets/DepositFacet.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
```

Comment:
seems a bit weird to me to have a deposit and a separate reward facet, let's merge thos

Reply:
Ok merged them